interface/weno/logsync.php

```diff
@@ -0,0 +1,117 @@
+<?php
```

Comment:
this file is always called from top of indexrx.php and called nowhere else. it already willhave a globals.php context and recalling that can mess things up. Much better to simply migrate all the code in this script into the indexrx.php script.

Reply:
Merged it into the class instead.